src/metrics.ts

```diff
@@ -278,6 +278,22 @@ export const getDataStreamSuccessesTotal = new promClient.Counter({
   labelNames: ['class', 'peer'] as const,
 });
 
+//
+// Chunk source metrics
+//
+
+export const requestChunkFromPeerTotal = new promClient.Counter({
+  name: 'request_chunk_total',
+  help: 'Count of each individual chunk http request, status can be error or success',
+  labelNames: ['status', 'class', 'method', 'peer'] as const,
```

Comment:
Let's remove `FromPeer` from the variable name, change the `peer` label to `source` (similar to the get data metrics), add a `type` that's either `trusted` or `peer`, and use this to count requests to the trusted node as well.

Reply:
pushed

I'm counting this metric on getChunkByAny method for trusted node, using the url stored in this.trustedNodeUrl as source.